crates/mock/src/utils.rs

```diff
@@ -0,0 +1,238 @@
+) {
+    let data = parser.parse(&sub_account_update).await;
+
+    assert!(data.is_ok(), "Error parsing account");
+
+    let data = data.unwrap();
+}
+
+pub fn run_async_task(callback: impl std::future::Future<Output = ()>) {
+    tokio::runtime::Builder::new_current_thread()
+        .enable_all()
+        .build()
+        .unwrap()
+        .block_on(callback);
+}
+
+pub fn get_subscribe_update_account(pubkey: &str, account: AccountInfo) -> SubscribeUpdateAccount {
+    let account = to_account(account);
+    let pubkey = Pubkey::from_str(pubkey).unwrap();
+    let subscriber_account_info = SubscribeUpdateAccountInfo {
+        pubkey: pubkey.as_ref().to_owned(),
+        lamports: account.lamports,
+        data: account.data.to_vec(),
+        owner: account.owner.as_ref().to_owned(),
+        executable: account.executable,
+        rent_epoch: account.rent_epoch,
+        write_version: 0,
+        txn_signature: None,
+    };
+
+    SubscribeUpdateAccount {
+        account: Some(subscriber_account_info),
+        slot: 0,
+        is_startup: false,
+    }
+}
+
+pub fn to_account(account_info: AccountInfo) -> Account {
+    let account_data = account_info.data.first().unwrap();
+    let account_data = general_purpose::STANDARD.decode(account_data).unwrap();
+    Account {
+        lamports: account_info.lamports,
+        data: account_data,
+        owner: Pubkey::from_str(&account_info.owner).unwrap(),
+        executable: account_info.executable,
+        rent_epoch: account_info.rent_epoch,
+    }
+}
+
+pub fn check_or_create_fixtures_dir() {
+    let dir_exists = Path::new(FIXTURES_PATH).is_dir();
+    if !dir_exists {
+        std::fs::create_dir(FIXTURES_PATH).unwrap();
+    }
+}
+
+pub fn get_account_data_file_path(pubkey: &str, cluster: ClusterType) -> Option<String> {
+    let dir_exits = Path::new(FIXTURES_PATH).is_dir();
+    if !dir_exits {
+        return None;
+    }
+    let mut file_name = format_public_key(pubkey);
+    match cluster {
+        ClusterType::Devnet => file_name.push_str("-devnet"),
+        ClusterType::Testnet => file_name.push_str("-testnet"),
+        ClusterType::MainnetBeta => file_name.push_str("-mainnet"),
+        _ => file_name.push_str("-mainnet"),
+    }
+    file_name.push_str(".json");
+
+    let file_path = Path::new(FIXTURES_PATH).join(file_name);
+
+    return Some(file_path.to_str().unwrap().to_string());
+}
+
+pub fn check_account_exists_on_fixtures(pubkey: &str, cluster: ClusterType) -> bool {
+    let file_path = get_account_data_file_path(pubkey, cluster);
+    match file_path {
+        Some(path) => Path::new(&path).is_file(),
+        None => false,
+    }
+}
+
+pub fn read_from_file(file_path: &str) -> Option<AccountInfo> {
+    let file = std::fs::read(file_path);
+    match file {
+        Ok(data) => {
+            let account: AccountInfo = serde_json::from_slice(&data).unwrap();
+            Some(account)
+        }
+        Err(_) => None,
+    }
+}
```

Comment:
Look at using map and ok
```suggestion
pub fn read_from_file(file_path: &str) -> Option<AccountInfo> {
    std::fs::read(file_path)
      .map(|data| {
        serde_json::from_slice::<AccountInfo>(&data)
      }).ok()
}
```